config/default.go

```diff
@@ -102,15 +100,21 @@ MaxTxSizeForL1 = 131072
 		ClosingSignalsManagerWaitForCheckingGER = "10s"
 		ClosingSignalsManagerWaitForCheckingForcedBatches = "10s"
 		ForcedBatchesFinalityNumberOfBlocks = 64
-		TimestampResolution = "15s"
-		SenderAddress = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+		TimestampResolution = "15s"	
 		PrivateKeys = [{Path = "/pk/sequencer.keystore", Password = "testonly"}]
```

Comment:
Shouldn't this line be removed? 
Even more, shouldn't we remove from the default config, values like addresses? We can keep them in the environment files.